module/module.go

```diff
@@ -413,16 +413,18 @@ func (m *Module) Ready(ctx context.Context, req *pb.ReadyRequest) (*pb.ReadyResp
 	m.mu.Lock()
 	defer m.mu.Unlock()
 	m.parentAddr = req.GetParentAddress()
```

Comment:
[minor] should this line be inside the if block below too? or do we run into errors if `m.parentAddr` isn't set for "no parent mode" also?

Reply:
We don't run into errors either way, but I'll make the change